src/main/resources/db/migration/V1.7__Add_Java8features_deck.sql

```diff
@@ -0,0 +1,41 @@
+insert into deck (description, name, rating, category_id, user_id, syntax, created_by) values ('Questions regarding Features that were implemented in Java 8', 'Java 8 Features', 0, 1, 1, 'JAVA', 1);
+
+--SET @deck_id = last_insert_id();
+--SET @deck_id = (SELECT AUTO_INCREMENT FROM  INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'orlp' AND   TABLE_NAME   = 'deck');
+SET @deck_id = (SELECT max(deck_id) from deck where name = 'Java 8 Features');
```

Comment:
Please take into account that
`SET @deck_id`
is mySQL specific syntax, it will probably not work if one day you decide to change the database provider.

Reply:
Do you know if there is any workaround?

Reply:
Here is a possible workaround:
![image](https://user-images.githubusercontent.com/3871580/47957547-5c4c0180-dfc0-11e8-82bf-fd6ef6836cef.png)

Reply:
What if we have multiple decks with the same name?

Reply:
I thought about this way but it means we will have more queries to our database which will make our script slower. If you will approve a way without a variable - I can rework it